Directory.Packages.props

```diff
@@ -2,6 +2,7 @@
   <PropertyGroup>
     <ManagePackageVersionsCentrally>true</ManagePackageVersionsCentrally>
     <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
+    <TargetFramework>net6.0</TargetFramework>
```

Comment:
This should be `net7.0`

Reply:
Yep, I messed up pulling from the upstream. Fixed now.